src/layout/Timer.tsx

```diff
@@ -27,7 +28,7 @@ export function renderToSVG(
         case 4: shiftX = height / 0.71; break;
         default: throw new Error("Unexpected Fraction Length");
     }
-    const x = `${componentWidth - 6 - 0.85 * shiftX}px`;
+    const x = `${componentWidth - parseInt(sidePadding, 10) - 0.85 * shiftX}px`;
```

Comment:
Can we move this parsing out to basically where we import it? Otherwise it just unnecessarily gets run all the time.